components/PixTypography.vue

```diff
@@ -10,48 +10,128 @@ export default {
       type: String,
       required: true,
     },
+    scale: {
+      type: String,
+      required: true,
+    },
   },
 };
 </script>
 
 <style lang="scss" scoped>
-h1 {
-  font-family: $open-sans;
-  font-weight: 300;
-  font-size: 3.75rem;
-  line-height: 4.5rem;
-  letter-spacing: -0.04em;
+* {
   color: $black-90;
   margin: 0;
 }
 
-h2 {
+.pix-title-large {
   font-family: $open-sans;
-  font-weight: 600;
-  font-size: 1.5rem;
-  line-height: 2rem;
+  font-size: 2rem;
+  line-height: 2.5625rem;
+  letter-spacing: -0.04em;
+  font-weight: 500;
+}
+
+/* Tablet */
+@media (min-width: 768px) {
+  .pix-title-large {
+    font-size: 2.5rem;
+    line-height: 3.125rem;
+    letter-spacing: -0.04em;
+  }
+}
+
+/* Desktop */
+@media (min-width: 980px) {
+  .pix-title-large {
+    font-size: 3rem;
+    line-height: 3.76rem;
+    letter-spacing: -0.04em;
+  }
+}
+
+.pix-title-medium {
+  font-family: $open-sans;
+  font-size: 1.625rem;
+  line-height: 2.0625rem;
   letter-spacing: -0.02em;
-  color: $black-90;
-  margin: 0;
+  font-weight: 500;
 }
 
-h3 {
+@media (min-width: 768px) {
+  .pix-title-medium {
+    font-size: 2rem;
+    line-height: 2.5625rem;
+    letter-spacing: -0.04em;
+  }
+}
+
+@media (min-width: 980px) {
+  .pix-title-medium {
+    font-size: 2.25rem;
+    line-height: 2.875rem;
+    letter-spacing: -0.04em;
+  }
+}
+
+.pix-title-small {
   font-family: $open-sans;
-  font-weight: 700;
-  font-size: 1.25rem;
+  font-size: 1.375rem;
   line-height: 1.75rem;
-  letter-spacing: 0em;
-  color: $black-90;
-  margin: 0;
+  letter-spacing: -0.02em;
+  font-weight: 500;
 }
 
-p {
+@media (min-width: 768px) {
+  .pix-title-small {
+    font-size: 1.5rem;
+    line-height: 2rem;
+    letter-spacing: -0.02em;
+  }
+}
+
+@media (min-width: 980px) {
+  .pix-title-small {
+    font-size: 1.75rem;
+    line-height: 2.25rem;
+    letter-spacing: -0.02em;
+  }
+}
+
+.pix-title-extra-small {
+  font-family: $open-sans;
+  font-size: 1.25rem;
+  line-height: 1.5rem;
+  letter-spacing: -0.02em;
+  font-weight: 500;
+}
+
+.pix-body-large {
   font-family: $roboto;
```

Comment:
```suggestion
.pix-typography[class*="scale-body"] {
  font-family: $roboto;
```

On pourrait indiquer que toutes les scales `body` sont en `Roboto`.